jbmc/src/java_bytecode/java_object_factory.cpp

```diff
@@ -153,6 +154,7 @@ class java_object_factoryt
     const exprt &max_length_expr,
     const source_locationt &location);
 
+
```

Comment:
Accidental change?

Reply:
What's the accidental change here?

Reply:
Adding an extra blank line apparently unrelated to the rest of the commit?